src/components/ingest/Form.js

```diff
@@ -52,6 +53,27 @@ const Form = observer(() => {
     onDrop: files => setFiles(files)
   });
 
+  useEffect(() => {
+    if(!masterAbr) {
+      setDisableDrm(true);
+      setDisableClear(true);
+      return;
+    }
+
+    const parsedProfile = JSON.parse(masterAbr);
+    const playoutFormats = Object.keys(
+      parsedProfile &&
+      parsedProfile.default_profile &&
+      parsedProfile.default_profile.playout_formats || {}
+    );
+
+    const drm = playoutFormats.filter(formatName => !formatName.includes("clear"));
+    const clear = playoutFormats.find(formatName => formatName.includes("clear"));
```

Comment:
No need to change the PR since this works but for info the drm vs. clear can be determined from the drm key under the playout format:  if `drm` is null then it's clear

```
        "dash-clear": {
          "drm": null,
          "protocol": {
            "min_buffer_length": 2,
            "type": "ProtoDash"
          }
        },
        "dash-widevine": {
          "drm": {
            "content_id": "",
            "enc_scheme_name": "cenc",
            "license_servers": [],
            "type": "DrmWidevine"
          },
          "protocol": {
            "min_buffer_length": 2,
            "type": "ProtoDash"
          }
        },
```